test/test-readability.js

```diff
@@ -158,23 +158,23 @@ function runTestsWithItems(label, domGenerationFn, source, expectedContent, expe
     });
 
     it("should extract expected title", function() {
-      expect(expectedMetadata.title).eql(result.title);
+      expect(result.title).eql(expectedMetadata.title);
```

Comment:
In the testsuite, the metadata checks had their expected/actual values reversed.

Reply:
D'oh! Thanks for fixing.